### ISSUE TITLE
[bug] DRM_IOCTL_NVIDIA_GEM_ALLOC_NVKMS_MEMORY failed

### ISSUE THREAD

**Issue by GaZaTu** [tauri-apps/tauri#7910]
### Describe the bug

Since updating my arch system yesterday i cannot run any tauri application regardless of the tauri versions (just an empty window opens).
I already tried downgrading linux-zen and all nvidia packages but that did not change anything.

edit: I know this is most likely not really a an issue with tauri itself but it is odd that only tauri apps are affected. Everything else (Qt, Firefox, Games) works fine.

edit2: the .AppImage i built before this error started happening still works for some reason.

edit3: if i build an .AppImage now without changing anything it will also be broken.

### Platform and versions

```text
[✔] Environment
    - OS: EndeavourOS Rolling Release X64
    ✔ webkit2gtk-4.1: 2.42.1
    ✔ rsvg2: 2.57.0
    ✔ rustc: 1.70.0 (90c541806 2023-05-31)
    ✔ Cargo: 1.70.0 (ec8a8a0ca 2023-04-25)
    ✔ rustup: 1.26.0 (2023-05-04)
    ✔ Rust toolchain: stable-x86_64-unknown-linux-gnu (default)
    - node: 16.20.2
    - npm: 10.1.0

[-] Packages
    - tauri [RUST]: 2.0.0-alpha.8
    - tauri-build [RUST]: 2.0.0-alpha.4
    - wry [RUST]: 0.27.3
    - tao [RUST]: 0.18.3
    - @tauri-apps/api [NPM]: 2.0.0-alpha.3
    - @tauri-apps/cli [NPM]: 2.0.0-alpha.8

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../dist
    - devPath: http://127.0.0.1:5173/
    - framework: SolidJS
    - bundler: Rollup
```

### Stack trace

```text
src/nv_gbm.c:99: GBM-DRV error (nv_gbm_bo_create): DRM_IOCTL_NVIDIA_GEM_ALLOC_NVKMS_MEMORY failed (ret=-1)

Failed to create GBM buffer of size 1000x800: Invalid argument
src/nv_gbm.c:99: GBM-DRV error (nv_gbm_bo_create): DRM_IOCTL_NVIDIA_GEM_ALLOC_NVKMS_MEMORY failed (ret=-1)

Failed to create GBM buffer of size 1000x800: Invalid argument
src/nv_gbm.c:99: GBM-DRV error (nv_gbm_bo_create): DRM_IOCTL_NVIDIA_GEM_ALLOC_NVKMS_MEMORY failed (ret=-1)

Failed to create GBM buffer of size 1000x800: Invalid argument
Failed to create EGL images for DMABufs with file descriptors -1, -1 and -1
```

### Additional context

pacman.log: https://gist.github.com/GaZaTu/355c137900ae3bc1bbb5b371eece4aec

**Comment by GaZaTu** [comment]
ok i found the culprit: webkit2gtk or webkit2gtk-4.1

i downgraded both from 2.42.1 to 2.40.5 and now everything works again.

**Comment by russmatney** [comment]
I just hit this as well, just to share an alternate solution - using either `WEBKIT_DISABLE_DMABUF_RENDERER=1` or `__NV_PRIME_RENDER_OFFLOAD=1` when running tauri solved it with the 2.42.1 versions of webkit2gtk.
